packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -66,6 +67,38 @@ export class RemoteMediaGroup {
     return [...this.unpinnedRemoteMedia, ...this.pinnedRemoteMedia];
   }
 
+  /**
+   * fixCsis
+   *
+   */
+  public fixCsis(
+    remoteMediaCsis: {remoteMedia: RemoteMedia; csi?: number | undefined}[],
+    commit = true
+  ): void {
+    forEach(remoteMediaCsis, ({csi, remoteMedia}) => {
+      if (csi) {
+        if (!(this.pinnedRemoteMedia.indexOf(remoteMedia) >= 0)) {
+          const unpinId = this.unpinnedRemoteMedia.indexOf(remoteMedia);
+          this.unpinnedRemoteMedia.splice(unpinId, 1);
+          this.pinnedRemoteMedia.push(remoteMedia);
+        }
+        remoteMedia.sendMediaRequest(csi, false);
+      } else {
+        if (!(this.unpinnedRemoteMedia.indexOf(remoteMedia) >= 0)) {
```

Comment:
maybe it's just me, but I feel that having a condition `x === -1` or `x < 0` would be clearer than having a `!(x >= 0)`

Reply:
For me this reads better as I read it as if it is not in unpinned/pinned array